src/containers/Trip/Trip.js

```diff
@@ -70,17 +70,22 @@ class Trip extends Component {
         const { city, country, departure_date, return_date } = this.state.trip
         return(
             <div className='container mt-5'>
-                <div className='row'>
-                    <h1>{ city }, { country }</h1>
-                    <div className='col-1'></div>
-                    <div className='' style={{}}>
-                        <p style={{ margin:'0' }}><span>{ departure_date.format('L') }</span> - <span>{ return_date.format('L') }</span></p>
-                        <p style={{ textAlign:'center', margin:'0' }}>{ moment().endOf(departure_date).to(return_date) }</p>
+                <div className='row justify-content-between'>
+                    <div className='col-lg-4' style={{}}>
+                        <div>
+                            <h5>Trip Details</h5>
+                            <h1 style={{fontSize:'3.5rem'}}>{ city }, { country }</h1>
```

Comment:
An `h1` shouldn't be used as a visual child within an `h5`. The numeric value of these headings is their semantic importance. "Trip Details" is more important than "City, Country" and so it should have a smaller number.

Judging by the `WeatherCard` component, it looks like the `h1` should be an `h6`?

If your goal is to make "City, Country" heading larger than "Trip Details" (which is totally reasonable), it'd be better to use CSS.

Reply:
Question,
Would I have to change the `h1` to a `p` tag?

Reply:
@aionate0812 it's still semantically a header - you can switch it to an `h6` and that should be fine.

Reply:
I see, so have it as an `h6` but just make it bigger using css?

Reply:
Exactly!